frontend/src/features/permits/components/permit-list/BasePermitList.tsx

```diff
@@ -129,16 +136,35 @@ export const BasePermitList = ({
     enablePagination: true,
     enableBottomToolbar: true,
     renderEmptyRowsFallback: () => <NoRecordsFound />,
-    renderRowActions: useCallback((props: { row: MRT_Row<PermitListItem> }) => {
-      return (
-        <Box sx={{ display: "flex", justifyContent: "flex-end" }}>
-          <PermitRowOptions
-            isExpired={isExpired}
-            permitId={props.row.original.permitId}
-          />
-        </Box>
-      );
-    }, []),
+    renderRowActions: useCallback(
+      ({ row }: { row: MRT_Row<PermitListItem> }) => {
+        const isInactive =
+          hasPermitExpired(row.original.expiryDate) ||
+          isPermitInactive(row.original.permitStatus);
+        return (
+          <Box sx={{ display: "flex", justifyContent: "flex-end" }}>
+            {DoesUserHaveAuthGroup({
```

Comment:
Should we use the original ```shouldShowRowActions(idirUserDetails?.userAuthGroup)``` check (from IDIRPermitSearchResults) here instead? It also includes EO

Reply:
On that note, maybe we should extract that method (shouldShowRowActions) out into a helper file so it can be reused?

Reply:
I wouldn't say it is necessary because EO cannot even act as company; so they cannot even get into this page. Spec allows only Staff Admin and PPC Clerk to act as company. If things change, we could revisit this implementation.

Spec:
https://github.com/bcgov/onRouteBCSpecification/blob/main/Staff%20Search/Staff%20search%20for%20Company.feature#L131

Reply:
Ok sure, np